raiden/tests/utils/factories.py

```diff
@@ -28,20 +29,59 @@
     NettingChannelEndState,
     NettingChannelState,
     RouteState,
+    TokenNetworkState,
     TransactionExecutionStatus,
     message_identifier_from_prng,
 )
+from raiden.transfer.state_change import ContractReceiveChannelNew, ContractReceiveRouteNew
 from raiden.transfer.utils import hash_balance_data
 from raiden.utils import CanonicalIdentifier, privatekey_to_address, random_secret, sha3, typing
 from raiden.utils.signer import LocalSigner, Signer
 
-EMPTY = object()
+EMPTY = 'empty'
+GENERATE = 'generate'
+
+
+def _partial_dict(full_dict: typing.Dict, *args) -> typing.Dict:
+    return {key: full_dict[key] for key in args}
+
+
+class Properties:
+    """ Base class for all properties classes. """
+    DEFAULTS: typing.ClassVar['Properties'] = None
```

Comment:
nitpick: Import all the stuff from typing directly on top. i.e:

`from typing import Classvar, XXX, YYY`


 No reason to refer to them by module.varname since there is no collision